apps/agreement/contracts/Agreement.sol

```diff
@@ -124,7 +124,7 @@ contract Agreement is ILockManager, IAgreement, IArbitrable, IACLOracle, AragonA
 
```

Comment:
Let me know if I should pull out e73a814678b6705ef656fb5f50d939a0c71337d1 into a different PR—it's quite small so I didn't bother too much.

Reply:
Totally fine :)